tools/create-translation-issues.sh

```diff
@@ -68,7 +68,7 @@ We are getting ready for the ${RELEASE} release. No further changes to translata
 
 We would be happy if you updated the Jamulus software translations for **${LANG}** by **${DEADLINE}**.
 
-Please
+Please either [update the translations on Weblate](https://hosted.weblate.org/projects/jamulus/) or use git:
```

Comment:
Does Weblate itself make it clear what _process_ needs following with appropriate step by step guidance?

Reply:
It's very self-explanatory. You're presented with a list of documents (components), you click on one, then on your language in the next window, and then you see a big button that says "Translate". It's hard to go wrong.

Reply:
It's very self-explanatory. You're presented with a list of documents (components), you click on one, then on your language in the next window, and then you see a big button that says "Translate". It's hard to go wrong.

Reply:
Yeah, I just looked briefly and it does look really simple.  It might be worth a general comment along the lines of "If you need help, just ask by tagging @jamulussoftware/translators in the issue you're working on" or something.